src/Misc/layoutbin/en-US/strings.json

```diff
@@ -138,6 +138,8 @@
   "NotLinux": "This Agent version is built for Linux. Please install a correct build for your OS.",
   "NotOSX": "This Agent version is built for OSX. Please install a correct build for your OS.",
   "NotWindows": "This Agent version is built for Windows. Please install a correct build for your OS.",
+  "NTLMPassword": "Password",
+  "NTLMUsername": "Domain\\Username",
```

Comment:
...not sure if that will cause more questions than is worth. Might be better to leave it as you have it.